### PR TITLE
wms info implementation

### PR DESCRIPTION
**Pull request by RoProducts**
Implementation for #11 

Note there are tests for generic behaviour inside the map module and a test for destination specific behaviour in the app module.

### CODE REVIEW

**Comment by Gnafu** [comment]
The getfeatureinfo_config.js file in the map module has info about SIIG specific application: please move it in the SIIG app module.
You can add an example "getfeatureinfo_config.js" in the map module with generic demo data.

**Comment by Gnafu** [comment]
If the GetFeatureInfo answer has multiple results, the result page has an arrow to navigate results, that's correct.
Clicking the arrow triggers another WMS request instead of showing the next result.

**Comment by RoProducts** [comment]
Fixed. Note the solution for tests for single flavours.

File: map/src/main/java/it/geosolutions/android/map/fragment/featureinfo/FeatureInfoAttributeListFragment.java [hunk 1]
```diff
@@ -202,7 +203,24 @@ private void startDataLoading(BaseFeatureInfoQuery query, ArrayList<Layer> layer
...
+	if(query instanceof WMSGetFeatureInfoQuery){
+        if(currentFeatures != null){
+            //start is the desired index
+            if (currentFeatures.size() > start) {
+                adapter.addAll(currentFeatures.get(start));
+
+                //setup the buttons manually
+
+                //if start > 0 previous is visible
+                getView().findViewById(R.id.previousButton).setVisibility(start > 0 ? View.VISIBLE : View.INVISIBLE);
```
Comment (Gnafu) [inline map/src/main/java/it/geosolutions/android/map/fragment/featureinfo/FeatureInfoAttributeListFragment.java]: findViewById() can return null.
To reproduce:
- query an area with multiple results
- rotate the device

**Comment by RoProducts** [comment]
Fixed issue on rotation change. Added a ProgressDialog in case of WMSGetFeatureInfoQuery.

**Pull request merged**